doc/release/1.2.0-notes.rst

```diff
@@ -47,9 +55,12 @@ Other changes
 Authors
 =======
 
+* Nicolas Hug +
+
 Issues closed
 -------------
 
+* `#6141 <https://github.com/scipy/scipy/issues/6141>`__: Request: transformation functions - Yeo-Johnson
```

Comment:
This and the authors list are generated during the release so you don't need to populate just now.